projects/js-packages/publicize-components/src/hooks/use-media-restrictions/index.js

```diff
@@ -137,10 +162,15 @@ const useMediaRestrictions = (
 					}
 					return errs;
 			  }, {} );
+
 		if ( JSON.stringify( newErrors ) !== JSON.stringify( errors.current ) ) {
 			errors.current = newErrors;
 		}
-		return errors.current;
+		return {
+			validationErrors: errors.current,
+			isConvertible:
+				isMediaConvertible( media.metaData ) && ! ( FILE_TYPE_ERROR in errors.current ), // Currently we cannot convert file-types.
```

Comment:
It looks like we could remove the file type error here, because we'll launch with file type conversions working.

I'm not sure this is working as expected either as I uploaded a WebP and got the conversion notice for Instagram.